packages/messages/src/use-cases/get-last-message-of-chat-room.js

```diff
@@ -0,0 +1,59 @@
+// Internal dependencies
+const logger = require('../utils/logger');
```

Comment:
Remeber to change the way we import logger

a) if you need to use Logger const { Logger } = require('@booking-services/shared');
b) if you need to reference logger file @booking-services/shared/src/utils/logger